core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -11,26 +11,38 @@
 
 package alluxio.master.meta.checkconf;
 
+import alluxio.Configuration;
 import alluxio.PropertyKey;
 import alluxio.PropertyKey.Scope;
 import alluxio.PropertyKey.ConsistencyCheckLevel;
 import alluxio.wire.Address;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 /**
  * This class is responsible for checking server-side configuration.
  */
 public class ServerConfigurationChecker {
+  private static final Logger LOG = LoggerFactory.getLogger(ServerConfigurationChecker.class);
+  private static final String HELP_INFO = "For details, please visit Alluxio web UI or"
+      + "run fsadmin doctor CLI.";
+  private static final long MAX_LOG_INTERVAL
```

Comment:
better to set this in the constructor. If we initialize it statically, this class will ignore tests which modify MASTER_CONFIG_REPORT_MAX_LOG_INTERVAL_MS